src/sidebar/components/ShareDialog/ShareDialog.tsx

```diff
@@ -9,23 +9,34 @@ import ShareAnnotations from './ShareAnnotations';
 import TabHeader from './TabHeader';
 import TabPanel from './TabPanel';
 
+export type ShareDialogProps = {
+  /** If true, the share tab will be rendered. Defaults to false */
+  shareTab?: boolean;
+  /** If true, the export tab will be rendered. Defaults to false */
+  exportTab?: boolean;
+  /** If true, the import tab will be rendered. Defaults to false */
+  importTab?: boolean;
```

Comment:
I considered prefixing these props with `show`, as in `showShareTab`, but I thought this would be clear enough.